doc/data/ruff.toml

```
@@ -5,9 +5,10 @@ line-length = 103
 extend-exclude = [
     "messages/d/duplicate-argument-name/bad.py",
     "messages/s/syntax-error/bad.py",
+    "messages/u/unreachable-match-patterns/bad.py",
     # syntax error in newer python versions
```

```suggestion
    # Syntax error in older python versions
    "messages/u/unreachable-match-patterns/bad.py",  # python 3.10
    # syntax error in newer python versions
```
Let's help ourselves for future cleanups

---

No you're right. But we don't support python 3.9 so let's not change anything